graphdatascience/pipeline/lp_training_pipeline.py

```diff
@@ -9,6 +9,16 @@
 
 class LPTrainingPipeline(ClassificationTrainingPipeline[LPModel]):
     def addFeature(self, feature_type: str, **config: Any) -> "Series[Any]":
+        """
+        Add a link feature to the pipeline.
+
+        Args:
+            feature_type: The type of feature to add. (For example, L2 or Hadamard)
```

Comment:
```suggestion
            feature_type: The type of feature to add. 
```
Would suggest to remove this to avoid getting this outdated